xchange-coindeal/src/main/java/org/knowm/xchange/coindeal/CoindealUtils.java

```diff
@@ -0,0 +1,11 @@
+package org.knowm.xchange.coindeal;
+
+import org.knowm.xchange.currency.CurrencyPair;
+
+public final class CoindealUtils {
+
+    public static String currencyPairToString(CurrencyPair currencyPair){
```

Comment:
this method could be in CoindealAdapters

Reply:
Fixed